### PR TITLE
ai001 find suitable dataset - done

### PR DESCRIPTION
**Pull request by Andrewwango**
Found 3 appropriate datasets to train on and compare. Also set up AI project structure in `/ai/` folder. Acceptance Criteria: Find a suitable dataset (done); Split the data into training, test and validation (taking an unsupervised approach so all data is used for training, and test data is a separate handmade testing dataset (explained in `/ai/01_eda.ipynb`)).

### CODE REVIEW

**Comment by Andrewwango** [comment]
Placeholder to ask for Liam's review

**Review by nataskainos — APPROVED** [review]
Great job! I have no comments.

File: ai/01_eda.ipynb [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+   "source": [
+    "## 1.1 Corpus datasets for training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8794cbcc-4c62-41c5-afeb-50c1bbb95a71",
+   "metadata": {},
+   "source": [
+    "The \"corpus\" datasets are used to train word embedding models (i.e. unsupervised learning) to be able to get vectors representing words. The similarity of vectors the n represent semantic association of words in the corpus. The corpuses are \"from the wild\" and should include examples of societal bias."
```
Comment (ldonnellykainoscom) [inline ai/01_eda.ipynb:34]: Should it be "the n vectors"

File: ai/01_eda.ipynb [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1e8f74f-4444-4781-80c9-c3d777715137",
+   "metadata": {},
+   "source": [
+    "The \"lexicon\" datasets are used to test the inferred vectors embedding certain words. We expect that gender-bias associated words from psychological/sociological studies to be close to gendered words in the learnt vector representations.\n",
+    "\n",
+    "**TODO** include all words + inflections in datasets"
```
Comment (ldonnellykainoscom) [inline ai/01_eda.ipynb:136]: Did you mean to leave this here? Probably best to not have any TODOs merged into main. I guess you can leave the inflections to the pre-processing ticket

**Comment by ldonnellykainoscom** [comment]
Looks really good! Interested to see where this goes. I like the references to the papers too.

Did you mean to leave the readme file empty? Or accidentally didn't commit the text?
If it's just empty, best to delete it 👍 

Same again for the empty or nearly empty notebooks. Ideally you want to only be merging 'production code' into main but ofc these are notebooks so not used in production but same principle of of committing working tidy code applies.

Other than that it looks tidy.

**Review by ldonnellykainoscom — CHANGES_REQUESTED** [review]

File: ai/01_eda.ipynb (hunk 2 above)
Comment (Andrewwango) [reply ai/01_eda.ipynb]: Hi, fixed

**Comment by Andrewwango** [comment]
> Looks really good! Interested to see where this goes. I like the references to the papers too.
> 
> Did you mean to leave the readme file empty? Or accidentally didn't commit the text? If it's just empty, best to delete it 👍
> 
> Same again for the empty or nearly empty notebooks. Ideally you want to only be merging 'production code' into main but ofc these are notebooks so not used in production but same principle of of committing working tidy code applies.
> 
> Other than that it looks tidy.

Cheers Liam, I didn't realise that was best practice, I've removed all non-essential notebooks to this ticket. And yeah README was just a placeholder before.

**Review by hellofadude — APPROVED** [review]
Approved!

**Pull request merged**